### ISSUE TITLE
Unhandled error at runner.go

### ISSUE THREAD

**Issue by GildedHonour** [reviewpad/reviewpad#44]
At `runner.go` there's:

        //........
	evalEnv.Collector.Collect("Completed Analysis", map[string]interface{}{
		"pullRequestUrl": evalEnv.PullRequest.URL,
	})

	return program, nil

However, the Collect can return an error:
```
type Collector interface {
	Collect(eventName string, properties map[string]interface{}) error
}
```

Therefore, there's an unhandled error for `evalEnv.Collector.Collect(...)` call

**Comment by marcelosousa** [comment]
Hi @GildedHonour - thanks for this contribution! 

Indeed, there's an unhandled error here.

**Comment by shay2025** [comment]
Fixed in #57
